src/lib/openapi/spec/api-token-schema.ts

```diff
@@ -12,6 +12,13 @@ export const apiTokenSchema = {
         },
         username: {
             type: 'string',
+            deprecated: true,
+        },
+        tokenName: {
+            type: 'string',
```

Comment:
Can it be both `null` and `undefined`? Why? 

I'd suggest having it always present on response schemas. On request schemas it could be optional and used instead of `username` if provided. Ideally we'd specify that you must **either** provide `username` or `tokenName` but that the other one could be missing. That requires some clever `oneOf` trickery, though.

Reply:
I guess of the two - `null` is the least likely one. On response it shouldn't be either, but for request it's either not set at all, or if a consumer has seen it and passed in a `null` which might not be very likely.

Yeah what to do during the migration phase of this property is an interesting discussion, but I'll set up tasks for that and we can discuss that there?

Reply:
Hmm, I'd set it to be non-nullable, I think. Because if the user provides a string for `username` but `null` for tokenname, which is suppose to supersede it: what do we do?